app/views/foods/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+  <%= render @food %>
+  <div>
+    <%= link_to "Back to foods", foods_path, class: "btn back-btn btn-outline-primary" %>
+    <%= button_to "Destroy this food", @food, method: :delete, class: "btn btn-outline-secondary" %>
+  </div>
```

Comment:
- [ ] Please make sure a food owner is the one able to delete a `food` currently I can delete a food without being its owner

Reply:
@DeliceLydia, that requirement would be for only a group of 2 members because in our UMD Diagram, User has no direct relationship with food:

![image](https://user-images.githubusercontent.com/52934128/218261327-cc8733c7-010f-4bc5-bcd3-6be1a7bab26c.png)